config/crd/kustomization.yaml

```diff
@@ -19,3 +19,9 @@ patchesStrategicMerge:
 # the following config is for teaching kustomize how to do kustomization for CRDs.
 configurations:
 - kustomizeconfig.yaml
+generatorOptions:
```

Comment:
Are the generatorOptions still in use?

Reply:
Actually since the addition of a separate kustomize run both options are now obsolete. I am going to remove them

Reply:
Done with 97ddd3f4